### PR TITLE
187651731 timeline slider UI

### PR DESCRIPTION
**Pull request by eireland**
Replaces timeline with a slider timeline.
This is the UI portion only. Slider thumb moves as time progresses, but does scrubbing thumb has no effect.

### CODE REVIEW

File: src/components/timeline/timeline.tsx [hunk 1]
```diff
@@ -1,19 +1,34 @@
-import React from "react";
+import React, { useEffect, useLayoutEffect, useState } from "react";
 import { observer } from "mobx-react";
 import { useStores } from "../../use-stores";
 import { VegetationStats } from "./vegetation-stats";
 import { FireEvents } from "./fire-events";
 
 import css from "./timeline.scss";
+import { Slider } from "@mui/material";
```
Comment (pjanik) [inline src/components/timeline/timeline.tsx:8]: We usually group regular imports before CSS imports.

File: src/components/timeline/timeline.tsx [hunk 2]
```diff
@@ -1,19 +1,34 @@
...
 
 const TICK_COUNT = 16;
 
 export const Timeline: React.FC = observer(function WrappedComponent() {
   const { simulation } = useStores();
+  const [val, setVal] = useState(simulation.timeInYears);
+  const [disabled, setDisabled] = useState(true);
```
Comment (pjanik) [inline src/components/timeline/timeline.tsx:15]: Assuming these variables are just temporary.

**Review by pjanik — APPROVED** [review]
It looks like a nice start. 👍 

When I play with the slider, it somehow feels sluggish, even though we don't do anything in its handler yet. Maybe it's the default animation. I'd try disabling it in the MUI Slider to see if it helps.

Note that currently, you can drag the handle past the current year, which we probably don't want. Also, you should double-check the UI specs, as I don't think we should be using any shadows.

**Comment by eireland** [comment]
Per UI spec, the slider thumb should have the same styling as the graph thumb which also has the shadows.
Limiting the thumb drag to last sim run year will be fixed in the next PR.

**Pull request merged**